proxy/ControlBase.cc

```diff
@@ -114,19 +113,19 @@ TimeMod::check(HttpRequestData *req) const
 TimeMod *
 TimeMod::make(char *value, const char **error)
 {
-  Tokenizer rangeTok("-");
```

Comment:
Are you trying to get rid of `Tokenizer`? How this change related to killing the `TsBuffer`?

Reply:
It's not directly related.  TsBuffer was used in this class, so I did some additional cleanup, replacing Tokenizer.  As I remember Tokenizer does memory allocation, which is avoided here by not using it.

Reply:
Would you make another PR for this change? Getting rid of Tokenizer seems good. But we easily miss something on this size of change.

Reply:
OK perhaps I will do that at some point when I have the time.

Reply:
It's not clear to me how to split this PR, without error-prone merge conflicts when merging the second PR.  It adds about 100 lines of new code.  Is that too big?

Reply:
I might be missing something, but my understanding is A). changes for `TextMod` and `MultiTextMod` are related to removing `TsBuffer`, I agree with that the changes should be done on this PR, but B) changes for `TimeMod` and `PortMod` are not related. 
How we get conflict with decoupling changes for `TimeMod` and `PortMod` from this PR?

Reply:
Alan asked me to remove Tokenizer:  https://github.com/apache/trafficserver/pull/7606#discussion_r602546817

Reply:
Both PRs would change ControlBase.cc.  I suppose it's possible none of changes would conflict.

Reply:
@SolidWallOfCode I'm not a big fan of getting rid of Tokenizer in *this* PR. I agree with it's a good change for performance. But it should be done by another PR after this PR is merged, IMO.